packages/fxa-geodb/test/maxmind-db-downloader.js

```diff
@@ -95,9 +95,9 @@ describe('maxmind-db-downloader', function () {
   describe('#integration - setupAutoUpdate', function () {
     it('auto update calls downloadAll correctly', function (done) {
       // test takes slightly over 5 seconds, set
-      // timeout to 6 seconds to ensure that we don't
+      // timeout to 10 seconds to ensure that we don't
```

Comment:
Did the CI changes make this take longer?

Reply:
I think this was just flakiness. Sometimes it would pass sometimes it wouldn't. As best I could tell this was just a response time on the download. Usually it would pass but I hit encountered this at least once where it took over 6 seconds.

I'll also add that since this PR touches the .circleci folder it means all the workspace packages get re-tested. I am pretty sure we rarely fxa-geodb, so this might have something to do with why we haven't noticed it until now.